### PR TITLE
DEV: Use DiscourseIpInfo for all IP queries.

### PR DESCRIPTION
**Pull request by nbianca**

### CODE REVIEW

File: config/locales/client.en.yml [hunk 1]
```diff
@@ -597,7 +597,7 @@ en:
       topics_entered: "topics entered"
       post_count: "# posts"
       confirm_delete_other_accounts: "Are you sure you want to delete these accounts?"
-      powered_by: "powered by <a href='https://ipinfo.io'>ipinfo.io</a>"
+      powered_by: "powered by <a href='https://maxmind.com'>maxmind.com</a>"
```
Comment (SamSaffron) [inline config/locales/client.en.yml]: I don't think we have any attribution requirements here, we can remove this imo. We do not want people to think we are uploading IPs to maxmind

File: lib/discourse_ip_info.rb [hunk 2]
```diff
@@ -16,27 +22,45 @@ def initialize
...
+    end
 
-    {
-      country: result.country.name,
-      country_code: result.country.iso_code,
-      region: result.subdivisions.most_specific.name,
-      city: result.city.name,
-    }
+    begin
+      result = Resolv::DNS.new.getname(ip)
```
Comment (SamSaffron) [inline lib/discourse_ip_info.rb]: Is this code ran in test, in general this is one of the super rare times I would be ok with mocking

**Comment by SamSaffron** [comment]
do we need any more tests here?

File: config/locales/client.en.yml (hunk 1 above)
Comment (nbianca) [reply config/locales/client.en.yml]: According to License section of [this document](https://dev.maxmind.com/geoip/legacy/geolite/), I believe we do.

Comment (ZogStriP) [reply config/locales/client.en.yml]: That only says we need to say so in ads and documentations ;)

**Pull request merged**